src/CSModule.sol

```diff
@@ -892,78 +900,18 @@ contract CSModule is
-                // `depositsLeft` is non-zero at this point all the time, so the check `depositsLeft > keysCount`
-                // covers the case when no depositable keys on the Node Operator have been left.
-                if (depositsLeft > keysCount || keysCount == keysInBatch) {
-                    // NOTE: `enqueuedCount` >= keysInBatch invariant should be checked.
-                    // @dev No need to safe cast due to internal logic
-                    no.enqueuedCount -= uint32(keysInBatch);
-                    // We've consumed all the keys in the batch, so we dequeue it.
-                    depositQueue.dequeue();
-                } else {
-                    // This branch covers the case when we stop in the middle of the batch.
-                    // We release the amount of keys consumed only, the rest will be kept.
-                    // @dev No need to safe cast due to internal logic
-                    no.enqueuedCount -= uint32(keysCount);
-                    // NOTE: `keysInBatch` can't be less than `keysCount` at this point.
-                    // We update the batch with the remaining keys.
-                    item = item.setKeys(keysInBatch - keysCount);
-                    // Store the updated batch back to the queue.
-                    depositQueue.queue[depositQueue.head] = item;
-                }
-
-                if (keysCount == 0) {
-                    continue;
-                }
-
-                // solhint-disable-next-line func-named-parameters
-                SigningKeys.loadKeysSigs(
-                    noId,
-                    no.totalDepositedKeys,
-                    keysCount,
-                    publicKeys,
-                    signatures,
-                    loadedKeysCount
-                );
-
-                // It's impossible in practice to reach the limit of these variables.
-                loadedKeysCount += keysCount;
-                // @dev No need to safe cast due to internal logic
-                no.totalDepositedKeys += uint32(keysCount);
-
-                emit DepositedSigningKeysCountChanged(
-                    noId,
-                    no.totalDepositedKeys
-                );
-
-                // No need for `_updateDepositableValidatorsCount` call since we update the number directly.
-                // `keysCount` is min of `depositableValidatorsCount` and `depositsLeft`.
-                // @dev No need to safe cast due to internal logic
-                uint32 newCount = no.depositableValidatorsCount -
-                    uint32(keysCount);
-                no.depositableValidatorsCount = newCount;
-                emit DepositableSigningKeysCountChanged(noId, newCount);
-                depositsLeft -= keysCount;
-                if (depositsLeft == 0) {
-                    break;
-                }
-            }
+        for (uint256 p = 0; p <= QUEUE_LOWEST_PRIORITY; ++p) {
+            // solhint-disable-next-line func-named-parameters
```

Comment:
Why disable? Just make them named. I think the byte code will not be affected

Reply:
For the same reason it's disabled here (the param names are basically the same as variables):

```
// solhint-disable-next-line func-named-parameters
SigningKeys.loadKeysSigs(
    noId,
    no.totalDepositedKeys,
    keysCount,
    publicKeys,
    signatures,
    loadedKeysCount
);
```